finality-aleph/src/sync/handler.rs

```diff
@@ -250,6 +250,11 @@ where
         Ok(maybe_id)
     }
 
+    /// Handle a single block.
+    pub fn handle_block(&mut self, block: B) {
+        self.block_importer.import_block(block)
```

Comment:
Uhhh, you don't want to just import any blocks that someone throws at you, you gotta check whether they are required, no?

Reply:
immensely reasonable, fixed